system-int-test/test/cmr/system_int_test/ingest/subscription_ingest_test.clj

```diff
@@ -472,3 +468,73 @@
 
           "Delete subscription as admin"
           ingest/delete-concept [concept2 {:token admin-user-token}] 200 nil)))))
+
+(deftest subscription-ingest-subscriber-collection-permission-check-test
```

Comment:
We are only testing user-type, not user's group id in this test. 
In general, we always test for both cases in terms of ACL permissions. I am debating if we have to do it for both cases every single test as eventually it goes down to the same function that check for user id based permissions.
I will leave it up to you if you want to add the ACL check based on user's group id.

Reply:
done